partcad/src/partcad/runtime_python_conda.py

```
@@ -79,7 +83,7 @@ def once_conda_locked(self):
                             "--json",
                             "-p",
                             self.path,
-                            "python=%s" % self.version,
+                            "python==%s" % self.version if self.is_mamba else "python=%s" % self.version,
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)